### PR TITLE
[#17854] [IOS] The tutorial video animation keeps playing after openi…

### LINKED ISSUE #17854: [IOS] The tutorial video animation keeps playing after opening a Status link
### Steps:
1. Send  the link into chat "http://status.im"
2. Open current link
3. Close it
4. Navigate to any other page in the Status app.

### Actual result:
The Status tutorial continues to play and follows you around the app, regardless of where you navigate.

https://github.com/status-im/status-mobile/assets/52490791/61b9895d-6e8c-4f3b-85e9-1d333dcb389a

### Expected result:
The tutorial should play until the current video is closed.

### Logs:
[geth.log](https://github.com/status-im/status-mobile/files/13298526/geth.log)
[Status.log](https://github.com/status-im/status-mobile/files/13298527/Status.log)

### ENV:
- iPhone 11 Pro max, IOS 16
- Nightly 8 Nov 2023

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes: #17854

### CODE REVIEW

File: src/status_im/ui/screens/browser/views.cljs [hunk 1]
```diff
@@ -225,7 +225,9 @@
                                                                          (.. ^js % -nativeEvent -data)])
         :on-load                                    #(re-frame/dispatch [:browser/loading-started])
         :on-error                                   #(re-frame/dispatch [:browser/error-occured])
-        :injected-java-script-before-content-loaded (js-res/ethereum-provider (str network-id))}])]
+        :injected-java-script-before-content-loaded (js-res/ethereum-provider (str network-id))
+        ;; https://github.com/status-im/status-mobile/issues/17854
+        :allowsInlineMediaPlayback                  true}])]
```
Comment (briansztamfater) [inline src/status_im/ui/screens/browser/views.cljs]: Can we use kebab case here?

**Review by briansztamfater — APPROVED** [review]

**Review by clauxx — APPROVED** [review]

File: src/status_im/ui/screens/browser/views.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/browser/views.cljs]: we can, but I'm not sure it's worth it :)

Comment (flexsurfer) [reply src/status_im/ui/screens/browser/views.cljs]: fixed

**Comment by VolodLytvynenko** [comment]
HI @flexsurfer Thank you for the fix. PR is tested and ready to be merged

**Review by alwx — APPROVED** [review]
thanks for handling that!

**Pull request merged**